### ISSUE TITLE
Shows two lines + some space characters when empty string or undefined model?

### ISSUE THREAD

**Issue by chrisspiegl** [vikingmute/vue3-easymde#1]
Hi, very cool port for vue3 and I really like using it.

However, I found that when I give the editor an empty string variable as a `v-model` it shows two lines and the second one shows a bunch of space like characters.

So nothing is visible, it's just two lines and I would rather not have those there?

Do you know a workaround or maybe a fix for this?

Thanks you,
Chris

**Comment by vikingmute** [comment]
Hey @chrisspiegl 
Could you provide me a repo or online link which could reproduce the issue?
So I can take a good look at it.

**Comment by chrisspiegl** [comment]
I created a example at: https://playcode.io/1314169

The main way to see the issue is to focus in the text area and do a <kbd>CMD</kbd> + <kbd>a</kbd> to select all and see the two lines and the whitespaces.

**A few notes:**

- The issue is present with either `ref(undefined)` or `ref('')`.
- It works as expected when the `ref('test')` string has any text in it.
- I found another issue (not sure why that is happening in the environment on playcode) with the `spellChecker` option which I had to set to `false` for it to work. Otherwise it throws an error of some kind of `CodeMirror.overlayMode` not being a function.
- And, the issue with the `undefiend` and `''` empty string is present in either `inputStyle: 'contenteditable'` or `inputStyle: 'textarea'`.

I hope this helps and makes the issue visible / reproducable.

Cheers 🌸
